packages/luvi/build.sh

```diff
@@ -17,36 +17,29 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 	-DWithPCRE2=On
 	-DWithSharedPCRE2=On
 	-DWithLPEG=On
+	-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src
+	-DLIBLUV_LIBRARIES=${TERMUX_PREFIX}/lib/libluv.so
```

Comment:
Generally speaking the headers and the library should come from the same place/the same version. Why use `-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src` and not `-DLIBLUV_INCLUDE_DIR=${TERMUX_PREFIX/include/luv`?

Reply:
I've tried to do that when trying to submit this package, it somewhat failed and I have no idea why. But I'll try to add it back.

Reply:
I think we risk issues if version that is stored in luvi's deps/luv/src/ deviates from version that is stored in $PREFIX/, would be great if we could use $TERMUX_PREFIX/include/luv instead, so please check what error it gives